src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -306,6 +307,8 @@ public static LocalizableErrorArgument Localize(this MessageID id)
             // Check for current experimental features, if any, in the current branch.
             switch (feature)
             {
+                case MessageID.IDS_DisableLengthBasedSwitch:
+                    return "disable-length-based-switch";
```

Comment:
It looks like this code path is only used from `TestOptions.WithExperimental(...)`. Consider removing this, and the associated `MessageID`, and replacing with a specific test helper, see `TestOptions.WithNoRefSafetyRulesAttribute()` for instance.